.travis.yml

```diff
@@ -12,7 +12,7 @@ script:
 - npm test
 - npm run build
 env:
-  - SITE_BASEURL_PRODUCTION="https://labs.usa.gov/" SITE_BASEURL_STAGING="https://ffd-microsite-staging.apps.cloud.gov/"
+  - SITE_BASEURL_PRODUCTION="https://ffd-microsite-production.apps.cloud.gov/" SITE_BASEURL_STAGING="https://ffd-microsite-staging.apps.cloud.gov/"
```

Comment:
Thanks, @donjo! Two questions:
1) Shouldn't these be `.app.cloud.gov`?
2) Are they meant to be on the same line?